README.md

```diff
@@ -91,3 +92,7 @@ We're happy to incorporate all improvements to JPlag into this code base. Feel f
 
 ### Adding new languages
 Adding a new language frontend is quite simple. Have a look at one of the `jplag.frontend` projects. All you need is a parser for the language (e.g., for ANTLR or for JavaCC) and a few lines of code that sends the tokens (that are generated by the parser) to JPlag.
+
+
+## License
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 0; Below](https://app.codacy.com/gh/DavidVarga1995/jplag/pullRequest?prid=8510190)
```suggestion
## License

```